cypress/support/elements/researcher_anonymous_classes_elements.js

```diff
@@ -0,0 +1,146 @@
+
+export const ResearcherClassesElements = {
+	getHeader() {
+		return cy.get("#primary h1");
+	},
+	verifyCohortsHeader() {
+		cy.get('#form-container form div span').eq(0).contains("Cohorts");
+	},
+	verifyTeachersHeader() {
+		cy.get('#form-container form div span').eq(2).contains("Teachers");
+	},
+	verifyResourcesHeader() {
+		cy.get('#form-container form div span').eq(4).contains("Resources");
+	},
+	getRemoveConcordConsortiumTeachersCheckbox() {
+		return cy.get('#form-container form [type=checkbox]');
+	},
+	verifyCheckboxText() {
+		cy.get('#form-container form [type=checkbox]').parent().should("contain", " Remove Concord Consortium Teachers? *");
+	},
+	verifyFooterNoteText() {
+		cy.get('#form-container form').should("contain", '* Concord Consortium Teachers belong to schools named "Concord Consortium".');
+	},
+	clickDropDown(dropDown) {
+		const option = ["Cohorts", "Teachers", "Resources"];
+		cy.get('#form-container form #react-select-'+(option.indexOf(dropDown)+2)+'-input').click({ force: true });
```

Comment:
I am locating the dropdown with the **id** locator. In the DOM, the **id** locator for the three dropdowns are given as **react-select-2-placeholder**, **react-select-3-placeholder**, **react-select-4-placeholder**. As you suggested in the previous PR comments, instead of duplicating the lines I am adding the index with 2 to locate the specific dropdown.